app/models/product.rb

```diff
@@ -66,16 +66,14 @@ def has_extension?
   end
 
   def mirror?
+    enabled = repositories.select(&:enabled)
```

Comment:
Just nitpicking, how about adding the `.to_a` here and remove it from the other lines?

Reply:
You are so very right. Updated!